### PR TITLE
Document sending notifications

### PR DESCRIPTION
**Pull request by mathjazz**
You can use Pontoon Stage to try it out.

I also ammended `.gitignore`.

@flodolo r?

### CODE REVIEW

File: tools/pontoon/sending_notifications.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Pontoon allows you to send in-app notifications to contributors of a specific project. To do so, go to your project dashboard and select the [Notifications](https://pontoon.mozilla.org/projects/pontoon-intro/notifications/) tab.
+
+## Choose recipient teams
+
+First you need to select project teams to send the notification to. You can select them manually or by using one of the available shortcuts:
+
+ * **All:** all teams translating the project
+ * **Complete:** teams with all strings translated in the project
+ * **Incomplete:** team that have untranslated strings in the project
```
Comment (flodolo) [inline tools/pontoon/sending_notifications.md]: team -> teams

File: tools/pontoon/sending_notifications.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Sending notifications in Pontoon
+
+Pontoon allows you to send in-app notifications to contributors of a specific project. To do so, go to your project dashboard and select the [Notifications](https://pontoon.mozilla.org/projects/pontoon-intro/notifications/) tab.
+
+## Choose recipient teams
+
+First you need to select project teams to send the notification to. You can select them manually or by using one of the available shortcuts:
+
+ * **All:** all teams translating the project
```
Comment (flodolo) [inline tools/pontoon/sending_notifications.md]: End list items with a period per https://github.com/mozilla-l10n/documentation/blob/master/misc/documentation_styleguide.md#typography

File: tools/pontoon/sending_notifications.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * **Complete:** teams with all strings translated in the project
+ * **Incomplete:** team that have untranslated strings in the project
+
+All project contributors from selected teams will receive the notification.
+
+You need to select at least one team.
+
+### Enter notification message
+
+Next, enter the message you'd like to send. You can use HTML tags. Try to be concise and use HTML wisely.
```
Comment (flodolo) [inline tools/pontoon/sending_notifications.md]: Please use proper apostrophes ’ instead of ' (same in next section)

**Comment by flodolo** [comment]
It looks good, mostly nits that I can fix directly if you prefer. Is there any plan for notifications to locales independently from projects?

**Review by flodolo — CHANGES_REQUESTED** [review]

**Comment by mathjazz** [comment]
> It looks good, mostly nits that I can fix directly if you prefer.

Updated and rebased.

> Is there any plan for notifications to locales independently from projects?

Yes. I'll send an email to `pontoon-team` to announce this and start the discussion on next steps.

**Review by flodolo — APPROVED** [review]

**Pull request merged**